pkgs/os-specific/linux/util-linux/default.nix

```
@@ -12,6 +12,9 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./rtcwake-search-PATH-for-shutdown.patch
+    # Remove patch below in 2.36.2, see https://github.com/karelzak/util-linux/issues/1193
+    (fetchpatch { url = "https://github.com/karelzak/util-linux/commit/52f730e47869ce630fafb24fd46f755dc7ffc691.patch";
+                  sha256 = "0lxmric30fqqb08cz225mvwpabas7rqp5w5h3m9z49pk0mybx4ja"; })
```

```suggestion
    # Remove patch below in 2.36.2, see https://github.com/karelzak/util-linux/issues/1193
    (fetchpatch {
      url = "https://github.com/karelzak/util-linux/commit/52f730e47869ce630fafb24fd46f755dc7ffc691.patch";
      sha256 = "0lxmric30fqqb08cz225mvwpabas7rqp5w5h3m9z49pk0mybx4ja";
    })
```

You need to add fetchpatch to inputs.